# FaceDetection.msg

uint8 num_faces    # Number of faces detected
Face[] faces       # Array of face data

